<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="4641" started-at="2019-01-10T00:26:17Z" finished-at="2019-01-10T00:26:22Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="4641" started-at="2019-01-10T00:26:17Z" finished-at="2019-01-10T00:26:22Z">
      <class name="FaruqAcademy.LoginTest">
        <test-method status="PASS" signature="getDriver()[pri:0, instance:FaruqAcademy.LoginTest@399f45b1]" name="getDriver" is-config="true" duration-ms="3853" started-at="2019-01-09T19:26:17Z" finished-at="2019-01-09T19:26:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getDriver -->
        <test-method status="FAIL" signature="loginTest()[pri:0, instance:FaruqAcademy.LoginTest@399f45b1]" name="loginTest" duration-ms="12" started-at="2019-01-09T19:26:21Z" finished-at="2019-01-09T19:26:21Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at FaruqAcademy.ExcelDataConfig.getData(ExcelDataConfig.java:24)
	at FaruqAcademy.LoginTest.loginTest(LoginTest.java:24)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:FaruqAcademy.LoginTest@399f45b1]" name="teardown" is-config="true" duration-ms="748" started-at="2019-01-09T19:26:21Z" finished-at="2019-01-09T19:26:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- FaruqAcademy.LoginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
